espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/exceptions/StaleElementException.java

```diff
@@ -0,0 +1,7 @@
+package io.appium.espressoserver.lib.handlers.exceptions;
+
+public class StaleElementException extends AppiumException {
+    public StaleElementException(String elementId) {
+        super(String.format("Element %s no longer exists", elementId));
```

Comment:
I like that. I would change it to View hierarchy, maybe? Correct me if I'm wrong, but accessibility objects are more of a UIAutomator thing right?